docs/data types.md

```diff
@@ -0,0 +1,277 @@
+---
+sidebar_position: 2
+---
+# Data Types
+
+*Written By: Watsamon Phongwanit*
+
+Code example can be run on: [https://runjs.co/](https://runjs.co/)
```

Comment:
Could be
You can run the code examples on:

Reply:
okie